front/notification_notificationtemplate.form.php

```diff
@@ -55,8 +55,11 @@
     Html::back();
 } else if (isset($_POST["purge"])) {
     $notiftpl->check($_POST["id"], PURGE);
+    $notiftpl->getFromDB($_POST['id']);
```

Comment:
is this line required ?

Reply:
I guess this is required to call `getLinkURL()` but this can be omitted if you use `getFormURLWithID()`.